File: BuilderDesignPattern/Desktop_Example/Driver.java
package BuilderDesignPattern.Desktop_Example;

import BuilderDesignPattern.Desktop_Example.Builders.DellDektopBuilder;
import BuilderDesignPattern.Desktop_Example.Builders.DesktopBuilder;
import BuilderDesignPattern.Desktop_Example.Builders.HpDesktopBuilder;
import BuilderDesignPattern.Desktop_Example.Directors.Director;
import BuilderDesignPattern.Desktop_Example.Products.Desktop;

public class Driver {
	public static void main(String[] args) {
		DesktopBuilder hpDesktopBuilder = new HpDesktopBuilder();
		DesktopBuilder dellDesktopBuilder = new DellDektopBuilder();

		Director director1 = new Director(hpDesktopBuilder);
		Director director2 = new Director(dellDesktopBuilder);

		Desktop desktop1 = director1.buildDeskstop();
		Desktop desktop2 = director2.buildDeskstop();

		System.out.println(desktop1);
		System.out.println(desktop2);
	}
}

File: BuilderDesignPattern/Destop_Example_Python/Driver.py
from BuilderDesignPattern.Destop_Example_Python.Builders.DELLDesktopBuilder import DELLDesktopBuilder
from BuilderDesignPattern.Destop_Example_Python.Builders.DesktopBuilder import DesktopBuilder
from BuilderDesignPattern.Destop_Example_Python.Builders.HPDesktopBuilder import HPDesktopBuilder
from BuilderDesignPattern.Destop_Example_Python.Director.Director import Director
from BuilderDesignPattern.Destop_Example_Python.Products.Desktop import Desktop


class Driver:
    def main(self):
        hpDesktopBuilder: DesktopBuilder = HPDesktopBuilder()
        dellDesktopBuilder: DesktopBuilder = DELLDesktopBuilder()

        director1: Director = Director(hpDesktopBuilder)
        director2: Director = Director(dellDesktopBuilder)

        desktop1: Desktop = director1.buildDesktop()
        desktop2: Desktop = director2.buildDesktop()

        print(desktop1)
        print(desktop2)

if __name__ == "__main__":
    driver: Driver = Driver()
    driver.main()